internal/global/trace.go

```diff
@@ -171,17 +171,14 @@ func (nonRecordingSpan) IsRecording() bool { return false }
 // SetStatus does nothing.
 func (nonRecordingSpan) SetStatus(codes.Code, string) {}
 
-// SetError does nothing.
-func (nonRecordingSpan) SetError(bool) {}
```

Comment:
I'm not sure why this method is being removed? It may be a cleanup. But then, I think it'd be better to do the removal in its own PR.

Reply:
Ah, sorry about that. I removed it as it looked like it was defunct, then decided I shouldn't be doing things like that in the same PR but didn't clean up all my clean-ups!